### ISSUE TITLE
Converting STP/STEP to x3d 

### ISSUE THREAD

**Issue by Callet91** [tpaviot/pythonocc-demos#41]
Hi there, 

I stumbled upon this repo when I was looking for a way to convert STP to x3d. I found some documentations [(https://pythonocc-doc.readthedocs.io/en/latest/convert/ )] but it seems that the code in this particular example is not longer supported and I can't find any other examples in this repo that shows a similiar procedure to convert STP to x3d. Is it still possible to use pythonocc to convert STP to x3d or is it not longer supported? 

Best regards, 
C

**Comment by tpaviot** [comment]
Yes it is, the code was updated. I have some material to upload to pythonocc-core
